app/eth2wrap/eth2wrap.go

```diff
@@ -305,3 +331,53 @@ func wrapError(ctx context.Context, err error, label string) error {
 
 	return errors.Wrap(err, "beacon api "+label, z.Str("label", label))
 }
+
+// newBestSelector returns a new bestSelector.
+func newBestSelector(n int, period time.Duration) *bestSelector {
+	return &bestSelector{
+		n:      n,
+		period: period,
+		counts: make([]int, n),
+	}
```

Comment:
nit since `Increment()` already does this
```suggestion
	    start: time.Now(),
	}
```

Reply:
sure, but better safe than sorry. You could technically do stuff with current time that will cause a panic if not initialised.